docs/tools/offline/cli.md

```diff
@@ -1,53 +1,42 @@
-### Mbed CLI
+## Mbed CLI
 
-Arm Mbed CLI is the name of the Arm Mbed command-line tool, packaged as `mbed-cli`. Mbed CLI enables Git- and Mercurial-based version control, dependencies management, code publishing, support for remotely hosted repositories (GitHub, GitLab and mbed.org), use of the Arm Mbed OS build system and export functions and other operations.
+Arm Mbed CLI is the name of the Arm Mbed command-line tool, packaged as `mbed-cli`. Mbed CLI enables Git- and Mercurial-based version control, dependencies management, code publishing, support for remotely hosted repositories (GitHub, GitLab and mbed.org), use of the Arm Mbed OS build system, export functions and other operations.
 
-This document covers the installation and usage of Mbed CLI.
+This document covers the installation and usage of Mbed CLI. You can find our legacy CLI documentation [here](https://github.com/ARMmbed/mbed-cli/blob/master/README.md).
 
-#### Using Mbed CLI
+## Installing and configuring
 
-The basic workflow for Mbed CLI is to:
```

Comment:
If the statement of functionality is removed (which might be fine as it's redundant with line 3), then perhaps line 3 should state "testing" as well.

Reply:
This may be covered in the "other operations" mentioned in that line. Do you think that most users will need testing called out in the opening paragraph?